api/pool.go

```diff
@@ -158,6 +159,15 @@ func removePoolHandler(w http.ResponseWriter, r *http.Request, t auth.Token) (er
 		return permission.ErrUnauthorized
 	}
 	poolName := r.URL.Query().Get(":name")
+	filter := &app.Filter{}
+	filter.Pool = poolName
+	apps, err := app.List(appFilterByContext([]permission.PermissionContext{}, filter))
+	if err != nil {
+		return err
+	}
+	if len(apps) > 0 {
+		return &terrors.HTTP{Code: http.StatusForbidden, Message: "This pool still have apps, before deleting It's needed to migrate or delete all apps"}
```

Comment:
I think "This pool has apps, you need to migrate or remove them before removing the pool" reads better. What do you think?